### ISSUE TITLE
Alpha Map

### ISSUE THREAD

**Issue by shadowlpd** [cgcostume/pubg-maps#9]
How did you know where the Parts of the Image must be set? Or can you program a Code so that the little Alpha Map pictures are converted to one Alpha Map picture, because in the folders are the Alpha Map Pictures too?
Thanks

**Comment by nevecex** [comment]
You have to rearrange them yourself like a puzzle so its random. Indices part points to location the first bracket 0 to 5 point to folder name ( etc. LS_03 ), and for example 60: (1, 3) tells that Texture2D_60.tga goes to spot 1 horizontally and 3 vertically from top, this is counted from 0 to 11. I'll see what I can rig up for alphas.

**Comment by shadowlpd** [comment]
Thank you very much

**Comment by shadowlpd** [comment]
![fortnite_athena_normal_rg8_lod0 2](https://user-images.githubusercontent.com/47274539/52289462-62120400-296e-11e9-9309-fd03d9548208.png)
![fortnite_athena_normal_rg8_lod0](https://user-images.githubusercontent.com/47274539/52289476-6807e500-296e-11e9-8be0-8d362f4dff1f.png)
I've find out some image-parts, which are at the right place, but it is very difficult to find out the right place for the image-parts. Of every image-part are two iamge-parts.

**Comment by nevecex** [comment]
After trying to figure this out I came to a realization this isn't alpha at all. This is the splat map. Splat map is used to map cliffs and grass on the map. Tiles you used in your first picture that look purplish green and are swirly are already used in height map and normal map so don't use those. Theres enough tiles for 3 images. 1 image is normal and height, and other two images are probably cliffs and grass. This is what I managed to do so far but its a pain in the a**. Also nice catch on the starter island tile missed that one. This one really is weird.
![fortnite_athena_cliff_lod0](https://user-images.githubusercontent.com/10382614/52290281-02b4f380-2970-11e9-8ee3-fd95eaaa1de4.png)
